File: app/src/main/java/com/xuvjso/nfmovies/Contract/LikedContract.java
package com.xuvjso.nfmovies.Contract;

import android.provider.BaseColumns;

public final class LikedContract {
    private LikedContract() {}

    public static class Liked implements BaseColumns {
        public static final String TABLE_NAME = "liked";
        public static final String COLUMN_NAME_NAME = "name";
        public static final String COLUMN_NAME_URL = "url";
        public static final String COLUMN_NAME_IMAGE = "image";
        public static final String COLUMN_NAME_SITE = "site";
    }
}

File: app/src/main/java/com/futuretech/nfmovies/Contract/LikedContract.kt
package com.futuretech.nfmovies.Contract

import android.provider.BaseColumns

object LikedContract {

    class Liked : BaseColumns {
        companion object {
            const val TABLE_NAME = "liked"
            const val COLUMN_NAME_NAME = "name"
            const val COLUMN_NAME_URL = "url"
            const val COLUMN_NAME_IMAGE = "image"
            const val COLUMN_NAME_SITE = "site"
            const val _ID = "id"
        }
    }
}
